packages/components/src/KaizenProvider/KaizenProvider.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { KaizenProvider, KaizenProviderProps } from "./KaizenProvider"
+
+const KaizenProviderWrapper = (props: KaizenProviderProps): JSX.Element => (
+  <KaizenProvider>{props?.children}</KaizenProvider>
+)
```

Comment:
You'd probably want something like this moving forward for testing the other props :)

```suggestion
const KaizenProviderWrapper = ({ children, ...restProps }: KaizenProviderProps): JSX.Element => (
  <KaizenProvider {...restProps}>{children}</KaizenProvider>
)
```